marketplace/deployer_util/config_helper.py

```diff
@@ -746,3 +756,10 @@ def _must_get_and_apply(dictionary, key, apply_fn, error_msg):
   """Similar to _maybe_get_and_apply but raises InvalidSchema if no such key."""
   value = _must_get(dictionary, key, error_msg)
   return apply_fn(value)
+
+
+def _must_contain_or_raise(value, valid_list, error_msg):
```

Comment:
drop `_or_raise` to be consistent with the other methods (it's also implied by these methods)